### DISCUSSION TITLE
How to click on second option of form dropdown selector not using self.select_option_by_text(dropdown_selector, option)

### DISCUSSION THREAD

**Discussion by zionkex** [seleniumbase/SeleniumBase#2545] [General]
Hello, Michael I have one little question, I have an element 
<select id="osvitaschedulereal-hometask_to" name="OsvitaScheduleReal[hometask_to]">
<option value="12446719">2024-03-14 (Поточний урок)</option>
<option value="12446720" selected="">2024-03-21</option>
<option value="12446721">2024-03-28</option>
</select> i want to click on second option.I use  driver.click("//select[@id='osvitaschedulereal-hometask_to']/option[2]"),now.Can you tell me another way how to click on second option

**Comment by mdmintz** [comment]
With the `SB()` and `BaseCase` formats, there are special methods for handling `<select>` elements:

```python
self.select_option_by_text(dropdown_selector, option, dropdown_by="css selector", timeout=None)
self.select_option_by_index(dropdown_selector, option, dropdown_by="css selector", timeout=None)
self.select_option_by_value(dropdown_selector, option, dropdown_by="css selector", timeout=None)
```

But for the `Driver()` format, use `driver.js_click(selector)` to click the element directly without opening the dropdown.

**Reply by tranngocminhhieu** [reply]
However, people want to use the Driver() format for several reasons. driver.js_click(selector) may not work. I hope Drive() can be powerful with all methods.

**Comment by zionkex** [comment]
> With the `SB()` and `BaseCase` formats, there are special methods for handling `<select>` elements:
> 
> ```python
> self.select_option_by_text(dropdown_selector, option, dropdown_by="css selector", timeout=None)
> self.select_option_by_index(dropdown_selector, option, dropdown_by="css selector", timeout=None)
> self.select_option_by_value(dropdown_selector, option, dropdown_by="css selector", timeout=None)
> ```
> 
> But for the `Driver()` format, use `driver.js_click(selector)` to click the element directly without opening the dropdown.
thank you
